scripts/setup_udev_rules.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+
+echo "Setting-up permissions for RealSense devices"
+
+exec 3>&2
+exec 2> /dev/null
+con_dev=$(ls /dev/video* | wc -l)
```

Comment:
This will find UVC but not T265. I can't thing of a generic way to have T265 boot/loaded devices other than explicit PID/VID

Reply:
We should also probably limit it to 8086 VID